back/app/models/idea_image.rb

```diff
@@ -33,6 +33,6 @@ class IdeaImage < ApplicationRecord
   def image_presence
     return if image.present?
 
-    errors.add(:image, 'cannot be blank!')
+    errors.add(:image, "can't be blank")
```

Comment:
Just making this new validation error (added recently in #3902) match with my preferred phrasing used in `layout_image` model also.